parts/kubernetesagentresourcesvmas.t

```diff
@@ -50,6 +51,7 @@
       },
       "type": "Microsoft.Storage/storageAccounts"
     },
+{{end}}
 {{if .HasDisks}}
```

Comment:
This looks like it can be added if ManagedDisks was specified.  It might help to follow the pattern in swarmagentresourcesvmas.t